Sustainsys.Saml2.StubIdp/App_Start/BundleConfig.cs

```diff
@@ -12,12 +11,10 @@ public static class BundleConfig
         public static void RegisterBundles(BundleCollection bundles)
         {
             bundles.Add(new Bundle("~/Content/css-bundle")
-                {
-                    Transforms = { new LessTransform() }
-                }
                 .Include(
                 "~/Content/css/select2.css",
-                "~/Content/site.less"));
+                "~/Content/site.css"
```

Comment:
dotless did not work and it seem to be used only for this one file compilation, so the site.less file was precompiled to site.css and dotless extension was removed